### PR TITLE
Clean up formats in quick-tour topics folder

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This was primarily as a fix for #23 (convert tables in quick-tour), but with reviewing the files in this folder I discovered there are no tables in any of these topics.

However, proceeded to clean up other formatting items and get these pages in better shape:

- Removed links from headings and the definition term items (in keeping with #17) and relocated to description text.
- Added alt text for images where needed
- Cleaned up a few areas where conditions for user guide editions was misapplied
- A few misc. formats to align with MD linting rules

## Affected documentation pages

These sections:

- https://docs.magento.com/m2/b2b/user_guide/getting-started/quick-tour.html
- (B2B only) https://docs.magento.com/m2/b2b/user_guide/quick-tour/b2b-quick-start.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Additional information

Internal staging build 227

### CODE REVIEW

File: src/quick-tour/customer-retention.md [hunk 1]
```diff
@@ -4,31 +4,30 @@ title: Retain Customers
...
-[Email Marketing Automation]({{ site.baseurl }}{% link marketing/engagement-cloud.md %})
-:  Send professionally designed, dynamic email campaigns with with live data from your Magento store, powered by [dotdigital][1]{: target="_blank"}
+Email Marketing Automation
+:  Send professionally designed, dynamic [email campaigns]({{ site.baseurl }}{% link marketing/engagement-cloud.md %}) with live data from your Magento store, powered by [dotdigital][1].
 
 <!--{% if "Default.EE-B2B" contains site.edition %}-->
-[RMA]({{ site.baseurl }}{% link sales/returns.md %})
-:  Customers can submit requests for Return Merchandise Authorization from your store. You can create shipment orders in a carrier system, and print shipping labels with RMA numbers.
+Merchandise Return
+:  Customers can submit requests for [Return Merchandise Authorization]({{ site.baseurl }}{% link sales/returns.md %}) from your store. You can create shipment orders in a carrier system, and print shipping labels with RMA numbers.
```
Comment (hickskenh) [inline src/quick-tour/customer-retention.md]: remove , after "carrier system"

File: src/quick-tour/differentiate-personalize.md [hunk 2]
```diff
@@ -3,20 +3,20 @@ conditions: Default.B2B Only
...
 
-[Smart Shopping Rules]({{ site.baseurl }}{% link marketing/promotions.md %})
-:  Customize the shopping experience with price rules and promotions that trigger at the product or shopping cart level.
+Smart Shopping Rules
+:  Customize the shopping experience with price rules and [promotions]({{ site.baseurl }}{% link marketing/promotions.md %}) that trigger at the product or shopping cart level.
 
-[Customer Groups]({{ site.baseurl }}{% link customers/customer-groups.md %})
-:  Offer different products and pricing according to customer group or [shared catalog]({{ site.baseurl }}{% link catalog/catalog-shared.md %}). Determine which discounts are available, and the tax class that applies to the order.
+Customer Groups
+:  Offer different products and pricing according to [customer group]({{ site.baseurl }}{% link customers/customer-groups.md %}) or [shared catalog]({{ site.baseurl }}{% link catalog/catalog-shared.md %}). Determine which discounts are available, and the tax class that applies to the order.
```
Comment (hickskenh) [inline src/quick-tour/differentiate-personalize.md]: remove , after "discounts are available"

File: src/quick-tour/differentiate-personalize.md [hunk 3]
```diff
@@ -3,20 +3,20 @@ conditions: Default.B2B Only
...
 
-[Maintain Multiple Tailored Websites]({{ site.baseurl }}{% link stores/websites-stores-views.md %})
-:  Maintain multiple tailored sites based on brand, geography, channel partner, or account. Introduce new market and languages, and track analytics from a single Admin.
+Maintain Multiple Tailored Websites
+:  Maintain [multiple tailored sites]({{ site.baseurl }}{% link stores/websites-stores-views.md %}) based on brand, geography, channel partner, or account. Introduce new market and languages, and track analytics from a single Admin.
```
Comment (hickskenh) [inline src/quick-tour/differentiate-personalize.md]: this looks like a series of 3.

"...or account, introduce new market and languages, and track analysis...."

File: src/quick-tour/engage.md [hunk 4]
```diff
@@ -4,25 +4,25 @@ title: Engage Your Customers
...
 
-[Content Management]({{ site.baseurl }}{% link cms/content-menu.md %})
+Content Management
 <!--{% if "Default.EE-B2B" contains site.edition %}-->
 :  Magento’s [Page Builder]({{ site.baseurl }}{% link cms/page-builder.md %}) makes it easy to create [targeted pages]({{ site.baseurl }}{% link cms/page-builder-add-dynamic-block.md %}) with [interactive elements]({{ site.baseurl }}{% link cms/page-builder-elements-buttons.md %}) that engage your customers. Even those without a technology background can create and manage content.
 <!--{% endif %}-->
 <!--{% if "Default.CE Only" contains site.edition %}-->
-:  Magento’s CMS makes it easy to store pages, or parts of pages, that you can use in your store. Even those without a technology background can create and manage site content.
+:  Magento’s [CMS]({{ site.baseurl }}{% link cms/content-menu.md %}) makes it easy to store pages, or parts of pages, that you can use in your store. Even those without a technology background can create and manage site content.
```
Comment (hickskenh) [inline src/quick-tour/engage.md]: replace "those" with "users"

File: src/quick-tour/increase-average-order-value.md [hunk 5]
```diff
@@ -4,28 +4,28 @@ title: Increase Average Order Value
...
+Email Reminders
+:  Send automated [reminder emails]({{ site.baseurl }}{% link marketing/email-reminder-rules.md %}) to customers who have added items to their carts or wish lists, but haven’t made a purchase. A variety of triggers can launch automated emails, including total cart value, quantity, items in the cart, and more.<!--{% endif %}-->
 
-[Full-Page Caching]({{ site.baseurl }}{% link system/cache-full-page.md %})
-:  Enhance performance by caching primary pages. Caching pages improves server response times, reduces load, and increases sustainable traffic. You can use tags to define which components to cache, so only relevant pages are cached as updates take place. It also has the ability to identify and differentiate visitors from shoppers.
+User Permissions and Roles
+:  [Restrict access]({{ site.baseurl }}{% link system/permissions.md %}) to data in the Admin on a “need to know” basis. Create multiple admin roles for read-only or editing privileges. Track and review all activity at a granular level to specific stores and websites.
```
Comment (hickskenh) [inline src/quick-tour/increase-average-order-value.md]: cap "Admin roles"?

File: src/quick-tour/increase-profitability.md [hunk 6]
```diff
@@ -3,17 +3,17 @@ conditions: Default.B2B Only
 title: Increase Profitability
 ---
 
-[Maintain Peak Performance][1]
-: With Magento Cloud hosting, your site is always optimized, and ready to scale up to meet demand on the biggest sales day of the year.
+Maintain Peak Performance
+: With [Magento Cloud hosting][1]{: target="_blank"}, your site is always optimized, and ready to scale up to meet demand on the biggest sales day of the year.
```
Comment (hickskenh) [inline src/quick-tour/increase-profitability.md]: remove , after optimized

File: src/quick-tour/loyalty-advocacy.md [hunk 7]
```diff
@@ -1,20 +1,20 @@
...
 
-[Advanced Reporting]({{ site.baseurl }}{% link reports/advanced-reporting.md %})
-:  Gain valuable insights at a glance with dynamic product, order, and customer reports, powered by Magento Business Intelligence.
+Advanced Reporting
+:  Gain valuable insights at a glance with dynamic product, order, and customer [reports]({{ site.baseurl }}{% link reports/advanced-reporting.md %}), powered by Magento Business Intelligence.
 
-[Dashboard Snapshots]({{ site.baseurl }}{% link stores/admin-dashboard.md %})
-:  Knowing what’s of interest on your site is crucial to maximize your marketing budget. Use this information to determine what you should cross- and up-sell to loyal customers, or which products to put on sale.
+Dashboard Snapshots
+:  Knowing what’s of interest on your site is crucial to maximize your marketing budget. Use this [information on your dashboard]({{ site.baseurl }}{% link stores/admin-dashboard.md %}) to determine what you should cross- and up-sell to loyal customers, or which products to put on sale.
```
Comment (hickskenh) [inline src/quick-tour/loyalty-advocacy.md]: remove , after customers

customers or which products to put on sale.

File: src/quick-tour/loyalty-advocacy.md [hunk 8]
```diff
@@ -1,20 +1,20 @@
...
 
-[Customer Accounts]({{ site.baseurl }}{% link customers/customer-account.md %})
-:  Opening an account provides customers with a personalized shopping experience that they can share with their friends. Customers can save their shopping preferences, and manage their own store billing and shipping information.
+Customer Accounts
+:  Opening an account provides [customers]({{ site.baseurl }}{% link customers/customer-account.md %}) with a personalized shopping experience that they can share with their friends. Customers can save their shopping preferences, and manage their own store billing and shipping information.
```
Comment (hickskenh) [inline src/quick-tour/loyalty-advocacy.md]: remove , after preferences

File: src/quick-tour/moment-of-purchase.md [hunk 9]
```diff
@@ -4,29 +4,26 @@ title: Moment of Purchase
...
 
-[Multiple Shipping Options]({{ site.baseurl }}{% link shipping/shipping.md %})
-:  Magento supports a variety of shipping methods so you can give your customers a choice at checkout. Customers can see a real-time estimate of shipping charges right from the shopping cart.
-
-[Shipping Labels]({{ site.baseurl }}{% link shipping/shipping-labels.md %})
-:  Merchants have complete control over package characteristics such as weight and size. Shipping labels, rate, and bar code information originates directly from the carrier. Labels can be generated for single or multiple orders.
+Shipping Labels
+:  Merchants have complete control over package characteristics such as weight and size. [Shipping labels]({{ site.baseurl }}{% link shipping/shipping-labels.md %}), rate, and bar code information originates directly from the carrier. Labels can be generated for single or multiple orders.
```
Comment (hickskenh) [inline src/quick-tour/moment-of-purchase.md]: might be too picky.  the last sentence.   should that be single- or multiple-item orders?

File: src/quick-tour/path-to-purchase.md [hunk 10]
```diff
@@ -2,21 +2,24 @@
...
-:  Your home page is like the front window display of your store. As the primary landing page, its design entices visitors to come inside for a closer look.
+Home Page
+:  Your [home page]({{ site.baseurl }}{% link quick-tour/home-page.md %}) is like the front window display of your store. As the primary landing page, its design entices visitors to come inside for a closer look.
 
-[Catalog Page]({{ site.baseurl }}{% link quick-tour/catalog-page.md %})
-:  This page shows products from your catalog in either a list or grid format. The selection can be based on a category chosen from the main menu, a choice made in the layered navigation on the left, or the results of a search. Any item can be examined in more detail, or placed directly into the shopping cart.
+Catalog Page
+:  The [Catalog page]({{ site.baseurl }}{% link quick-tour/catalog-page.md %}) shows products from your catalog in either a list or grid format. Customers can make selections  based on a category they choose from the main menu, by using the layered navigation on the left, or from the results of a search. They can examine the item in more detail, or placed it directly into the shopping cart.
```
Comment (hickskenh) [inline src/quick-tour/path-to-purchase.md]: 1. remove , after "left"
2. "more detail or place it directly".   remove , and "d"

File: src/quick-tour/reduce-ordering-errors.md [hunk 11]
```diff
@@ -3,23 +3,22 @@ conditions: Default.B2B Only
...
-reduces the process to several clicks. Magento offers multiple ways to configure the [shopping cart]({{ site.baseurl }}{% link sales/cart-configuration.md %}) and [checkout process]({{ site.baseurl }}{% link sales/checkout-configuration.md %}), to eliminate manual data entry, and reduce costly ordering errors.
+Streamlined Order Process
+:  [Quick Order]({{ site.baseurl }}{% link sales/quick-order.md %}) reduces the process to several clicks. Magento offers multiple ways to configure the [shopping cart]({{ site.baseurl }}{% link sales/cart-configuration.md %}) and [checkout process]({{ site.baseurl }}{% link sales/checkout-configuration.md %}), to eliminate manual data entry and reduce costly ordering errors.
 
-[Assisted Sales]({{ site.baseurl }}{% link sales/shopping-assistance.md %})
-:  Assign a dedicated sales rep to each company account, who can access the customer’s shopping cart in real time, and offer personal assistance over the phone.
+Assisted Sales
+:  Assign a [dedicated sales rep]({{ site.baseurl }}{% link sales/shopping-assistance.md %}) to each company account, who can access the customer’s shopping cart in real time, and offer personal assistance over the phone.
```
Comment (hickskenh) [inline src/quick-tour/reduce-ordering-errors.md]: agent tasks need to come before account.

Assign a dedicated sales rep, who can access the customer’s shopping cart in real time and offer personal assistance over the phone, to each company account.

File: src/quick-tour/reduce-ordering-errors.md [hunk 12]
```diff
@@ -3,23 +3,22 @@ conditions: Default.B2B Only
...
 
-[Custom Catalogs and Price Lists]({{ site.baseurl }}{% link catalog/catalog-shared.md %})
-: Maintain different “shared” catalogs with custom pricing for specific companies.
+Custom Catalogs and Price Lists
+: Maintain different “shared” [catalogs with custom pricing]({{ site.baseurl }}{% link catalog/catalog-shared.md %}) for specific companies.
 
-[Bulk Orders]({{ site.baseurl }}{% link customers/account-dashboard-order-by-sku.md %})
-:  Create bulk pricing tiers with per-unit costs and discounts according to order size. Buyers can [reorder]({{ site.baseurl }}{% link customers/account-dashboard-my-orders.md %}) from previous orders, or upload order data directly to the shopping cart.
+Bulk Orders
+:  Create [bulk pricing tiers]({{ site.baseurl }}{% link customers/account-dashboard-order-by-sku.md %}) with per-unit costs and discounts according to order size. Buyers can [reorder]({{ site.baseurl }}{% link customers/account-dashboard-my-orders.md %}) from previous orders, or upload order data directly to the shopping cart.
```
Comment (hickskenh) [inline src/quick-tour/reduce-ordering-errors.md]: remove , after orders

File: src/quick-tour/self-service-tools.md [hunk 13]
```diff
@@ -3,23 +3,23 @@ conditions: Default.B2B Only
 title: Empower with Self-Service Tools
 ---
 
-[Self-Service Tools]({{ site.baseurl }}{% link customers/account-dashboard.md %})
-:  Magento’s self-service model empowers customers to manage their own accounts and ordering process.
+Self-Service Tools
+:  Magento’s self-service model provides an [account dashboard]({{ site.baseurl }}{% link customers/account-dashboard.md %}) that empowers customers to manage their own accounts and ordering process.
```
Comment (hickskenh) [inline src/quick-tour/self-service-tools.md]: remove "that"?

File: src/quick-tour/attract-customers.md [hunk 14]
```diff
@@ -4,18 +4,17 @@ title: Attract New Customers
 
 Magento is packed with features that make it easy to create a “search engine friendly” websites and increase the likelihood of bringing the right customers to your site.
 
-[Search Engine Optimization]({{ site.baseurl }}{% link marketing/seo-search.md %})
-:  Magento offers powerful, native capabilities to streamline Search Engine Optimization (SEO) practices for content and site exposure that are integrated with the Admin, and tied directly into the user experience.
+Search Engine Optimization
```
Comment (shrielenee) [inline src/quick-tour/attract-customers.md]: I would make these headings, or all of the content in a markdown table. This formatting is a little strange.

File: src/quick-tour/build-loyalty.md [hunk 15]
```diff
@@ -3,20 +3,20 @@ conditions: Default.B2B Only
 title: Build Loyalty
 ---
 
-[Friction-Free Purchasing]({{ site.baseurl }}{% link customers/account-dashboard.md %})
-:  Magento’s self-service model makes it easy to build loyalty for fast, friction-free purchasing.
+Friction-Free Purchasing
```
Comment (shrielenee) [inline src/quick-tour/build-loyalty.md]: Same here

File: src/quick-tour/customer-journey.md [hunk 16]
```diff
@@ -2,26 +2,20 @@
 title: Customer Journey
 ---
 
-## Attract New Customers
+Attract New Customers
```
Comment (shrielenee) [inline src/quick-tour/customer-journey.md]: Same here

File: src/quick-tour/customer-retention.md [hunk 17]
```diff
@@ -4,31 +4,30 @@ title: Retain Customers
 
 Magento makes it easy for you to get repeat business and build brand loyalty. Magento gives you total control and flexibility over creating and revising goodies like rewards programs, custom coupons and automated emails to keep your customers coming back again and again.
 
-[Email Marketing Automation]({{ site.baseurl }}{% link marketing/engagement-cloud.md %})
-:  Send professionally designed, dynamic email campaigns with with live data from your Magento store, powered by [dotdigital][1]{: target="_blank"}
+Email Marketing Automation
```
Comment (shrielenee) [inline src/quick-tour/customer-retention.md]: Same here

File: src/quick-tour/differentiate-personalize.md [hunk 18]
```diff
@@ -3,20 +3,20 @@ conditions: Default.B2B Only
 title: Differentiate and Personalize
 ---
 
-[Create Custom Experiences]({{ site.baseurl }}{% link quick-tour/engage.md %})
-:  Magento offers a rich set of tools to create personalized experiences across multiple digital touchpoints, based on customer activity and profile.
+Create Custom Experiences
```
Comment (shrielenee) [inline src/quick-tour/differentiate-personalize.md]: Same here, I find this formatting a bit clunky

File: src/quick-tour/engage.md [hunk 19]
```diff
@@ -4,25 +4,25 @@ title: Engage Your Customers
 
 Magento makes it easy to create a customized, engaging site experience. Encourage your customers to spend more time exploring your site, and give them the tools to make it easy to find what they want faster.
 
-[Content Management]({{ site.baseurl }}{% link cms/content-menu.md %})
+Content Management
```
Comment (shrielenee) [inline src/quick-tour/engage.md]: See above

File: src/quick-tour/increase-average-order-value.md [hunk 20]
```diff
@@ -4,28 +4,28 @@ title: Increase Average Order Value
 
 Magento provides a range of tools to help you tailor the shopping experience, and encourage your customers to put more items in their shopping carts and spend more money.
 
-[Targeted Promotions]({{ site.baseurl }}{% link marketing/promotions.md %})
-:  Use catalog and shopping cart price rules to create promotions that kick into gear when a set of conditions is met. {% if "Default.EE-B2B" contains site.edition %}Segment customers dynamically and build segments based on specific characteristics such as customer address, order history, shopping cart content, and much more. Then, use [Page Builder]({{ site.baseurl }}{% link cms/page-builder.md %}) to create a [dynamic block]({{ site.baseurl }}{% link cms/dynamic-blocks.md %}) that is triggered by a promotion, and appears only to the targeted customer segment.{% endif %}
+Targeted Promotions
```
Comment (shrielenee) [inline src/quick-tour/increase-average-order-value.md]: Same as my previous remarks. Would be nice to have these as headings so they're available in the right-hand TOC.

File: src/quick-tour/increase-profitability.md [hunk 21]
```diff
@@ -3,17 +3,17 @@ conditions: Default.B2B Only
 title: Increase Profitability
 ---
 
-[Maintain Peak Performance][1]
-: With Magento Cloud hosting, your site is always optimized, and ready to scale up to meet demand on the biggest sales day of the year.
+Maintain Peak Performance
```
Comment (shrielenee) [inline src/quick-tour/increase-profitability.md]: Recommend headings

Comment (shrielenee) [reply src/quick-tour/increase-profitability.md]: My suggestion is applicable to all following pages with similar formatting

**Comment by leslietilling** [comment]
The term-definition formatting was put in at the conversion to more closely align with the original page intentions. These are sort of overview pages that provide benefit statements that map to the specific product features. We will likely re-vamp this content in the future, but for now I think it works better in this format over an H2 because each item is so small.

**Comment by leslietilling** [comment]
running tests

File: src/quick-tour/increase-average-order-value.md (hunk 5 above)
Comment (leslietilling) [reply src/quick-tour/increase-average-order-value.md]: not in this case.... not really a proper noun here

File: src/quick-tour/moment-of-purchase.md (hunk 9 above)
Comment (leslietilling) [reply src/quick-tour/moment-of-purchase.md]: rewrote... that isn't what it means but I made it clearer.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**